include/openssl/asn1t.h

```diff
@@ -715,13 +715,16 @@ typedef struct ASN1_PRIMITIVE_FUNCS_st {
 
 typedef int ASN1_aux_cb(int operation, ASN1_VALUE **in, const ASN1_ITEM *it,
                         void *exarg);
+typedef int ASN1_aux_const_cb(int operation, const ASN1_VALUE **in, const ASN1_ITEM *it,
+                        void *exarg);
 
 typedef struct ASN1_AUX_st {
     void *app_data;
     int flags;
     int ref_offset;             /* Offset of reference value */
     int ref_lock;               /* Lock type to use */
     ASN1_aux_cb *asn1_cb;
+    ASN1_aux_const_cb *asn1_const_cb;
     int enc_offset;             /* Offset of ASN1_ENCODING structure */
 } ASN1_AUX;
```

Comment:
Since this is a non-opaque type we should probably add new fields to the end of the structure rather than in the middle. I'm thinking that, in the event that end user applications use this type directly, then any static initialisers will still continue to work as before, i.e. all the old fields are still there and in the same order.

Reply:
Glad that you're continuing your review.

I agree that for maximal backward compatibility new structure elements should better be added at its end.  I've just changed this accordingly (although conceptually it would have been nicer to keep `asn1_const_cb` just next to its sibling` asn1_cb`).